pkg/installmanager/installmanager.go

```diff
@@ -796,7 +825,7 @@ func (m *InstallManager) initSSHKey() (string, error) {
 
 }
 
-func (m *InstallManager) initSSHAgent(sshKeyPath string) (func(), error) {
+func (m *InstallManager) initSSHAgent(sshKeyPaths ...string) (func(), error) {
```

Comment:
Is this function variadic for any reason other than not wanting to change the tests to pass it a slice?

I would prefer to see this as `initSSHAgent(sshKeyPaths []string)` unless we really want the variadic feature.

Reply:
Yeah just less test churn and seemed a little more concise. Updating to use slice, but what was the motivation here, just to avoid unexpected things if someone ever adds another arg?